tools/oss-fuzz/fuzz_ipc.c

```diff
@@ -22,7 +23,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
 	// copy the buffer to pre allocated buffer
 	struct sof_ipc_cmd_hdr *hdr = malloc(SOF_IPC_MSG_MAX_SIZE);
 
-	memcpy_s(hdr, Data, min(Size, SOF_IPC_MSG_MAX_SIZE));
+	memcpy_s(hdr, SOF_IPC_MSG_MAX_SIZE, Data, MIN(Size, SOF_IPC_MSG_MAX_SIZE));
```

Comment:
Should this be a PR for v1.6 fix ?

Reply:
This is in the fuzzers stub itself, I will squash this fix into the original commit to add the fuzzers. It has no effect on the firmware.